sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -263,6 +267,7 @@ public partial class ServiceBusProcessorOptions
     {
         public ServiceBusProcessorOptions() { }
         public bool AutoCompleteMessages { get { throw null; } set { } }
+        public string Identifier { get { throw null; } set { } }
```

Comment:
I think we will also want to update ServiceBusSessionProcessorOptions and ServiceBusSessionProcessor.

Reply:
@JoshLove-msft should I also do the same for the receiver and sender? `ServiceBusSessionReceiver`, `ServiceBusSessionReceiverOptions`, etc?

Reply:
If they don't inherit directly from the non-session types (where they'd get the Identifier for free), we should include them.